source/standards/penetration-testing-services.html.md.erb

```diff
@@ -0,0 +1,67 @@
+---
+title: Testing Services
+expires: 2018-01-10
+---
+
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+## Why and when to test
+
+You should carry out [penetration testing](https://www.gov.uk/service-manual/technology/vulnerability-and-penetration-testing) and vulnerability assessments to ensure your service is secure, according to the [Service Manual](https://www.gov.uk/service-manual):
+
+> "Vulnerability assessments help you find potential weaknesses in your service.
+> Penetration tests proactively attack your systems to find weaknesses and help you understand how easy they are to exploit."
+
+At a minimum you should test twice a year at six month intervals, your tests should consist of:
+
+* one external test by an [approved third party](https://www.gov.uk/service-manual/technology/vulnerability-and-penetration-testing#working-with-third-parties) which is classed as a major test
+* one internal test, which is classed as a minor test, carried out by members of the GDS Security Team
+
+In addition to routine testing every 6 months, you should also test following significant changes to your infrastructure or network, for example when you:
+
+* change a cloud service provider
+* change or upgrade an operating system
+* make significant changes to your firewall configuration
+
+You should also seek clarification from IA whether your service is classed as large or minor in terms of potential risk. This could affect how frequently you need to test as large services may need to test more frequently than twice a year.
+
+Test frequency also depends on the complexity of your service:
+
+* a complex service is not necessarily defined as a large service
+* smaller services can be considered as complex depending on their specific configuration
+
+IA can provide clarification on whether your service is considered as large or small in terms of its complexity. You will need to work with IA to establish a testing pattern and meet with them regularly to discuss ongoing changes to your service.
+
+## How to test?
+
+The costs involved in testing can be significant so it is advisable to use GDS internal services as far as possible.
+
+While, it is not practical to test all components of a service every six months; whether the test is a major external test or a minor internal test will determine which components of your service to focus testing on.
+
+Before you begin testing clearly define:
+
+* the beginning and end test states
+* the explicit results you want and do not want to achieve
+* the specific technical requirements to complete your testing
+
+If your testing produces negative results:
```

Comment:
I read "negative results" as meaning "no problems were found" but it looks like you mean "problems were found"?

Reply:
looks like this also confused @davbo 😄